tests/src/python/test_qgscheckablecombobox.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+"""QGIS Unit tests for QgsCheckableComboBox
+
+.. note:: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 2 of the License, or
+(at your option) any later version.
+"""
+__author__ = 'Alexander Bruy'
+__date__ = '22/03/2017'
+__copyright__ = 'Copyright 2017, The QGIS Project'
+# This will get replaced with a git SHA1 when you do a git archive
+__revision__ = '$Format:%H$'
+
+import qgis  # NOQA
+
+from qgis.PyQt.QtCore import Qt
+from qgis.gui import QgsCheckableComboBox
+
+try:
```

Comment:
This shouldn't be required anymore - it was only needed when we still supported Qt4 and QSignalSpy wasn't available. It's always present for Qt5 builds